Proofreading fixes 0x04i Testing-User-Privacy-Protection.md

thornshadow99 opened the pull request:
This is an edit of the Mobile App User Privacy document by technical writer Rob Thornton. It tightens up the writing without significantly changing the document.

cpholguera commented on Document/0x04i-Testing-User-Privacy-Protection.md:
```diff
@@ -2,72 +2,72 @@
+### The New Approach (Google and Apple)
 
-In order to address these challenges and help users easily understand how their data is being collected, handled, and shared, Google and Apple introduced new privacy labeling systems (very much along the lines of NIST's proposal for [Consumer Software Cybersecurity Labeling](https://nvlpubs.nist.gov/nistpubs/CSWP/NIST.CSWP.02042022-1.pdf):
+In order to address these challenge and better inform users, Google and Apple have introduced new privacy labeling systems (very much along the lines of NIST's proposal) to help users easily understand how their data is being collected, handled, and shared, [Consumer Software Cybersecurity Labeling](https://nvlpubs.nist.gov/nistpubs/CSWP/NIST.CSWP.02042022-1.pdf). Their approaches can be seen at: ...
```
Markdown lint fix.
```suggestion
In order to address these challenge and better inform users, Google and Apple have introduced new privacy labeling systems (very much along the lines of NIST's proposal) to help users easily understand how their data is being collected, handled, and shared, [Consumer Software Cybersecurity Labeling](https://nvlpubs.nist.gov/nistpubs/CSWP/NIST.CSWP.02042022-1.pdf). Their approaches can be seen at:
```

cpholguera commented on Document/0x04i-Testing-User-Privacy-Protection.md:
```diff
@@ -2,72 +2,72 @@
+### Google ADA MASA Program
 
-Performing regular security testing can help developers identify key vulnerabilities in their apps. Google Play will allow developers who have completed independent security validation to showcase this on their Data safety section. This helps users feel more confident about an app's commitment to security and privacy.
+Since regular security testing helps developers identify key vulnerabilities in their apps, Google Play will allow developers who have completed independent security validation to inform users by disclosing this fact in the app's Data Safety section. The developer's commitment to security and privacy is meant to reassure users. 
```
Markdown lint fix.
```suggestion
Since regular security testing helps developers identify key vulnerabilities in their apps, Google Play will allow developers who have completed independent security validation to inform users by disclosing this fact in the app's Data Safety section. The developer's commitment to security and privacy is meant to reassure users.
```

cpholguera reviewed (approved):
This is really good, thank you @thornshadow99 and apologies on behalf of the project team for the delay! We've been focusing on the [new proposals, core updates and changes](https://mas.owasp.org/news/) to be able to launch the new MASTG refactoring this year.


We hope to have more contributions from you this year. Thank you very much!

cpholguera commented on Document/0x04i-Testing-User-Privacy-Protection.md:
```diff
@@ -2,72 +2,72 @@
+### The New Approach (Google and Apple)
 
-In order to address these challenges and help users easily understand how their data is being collected, handled, and shared, Google and Apple introduced new privacy labeling systems (very much along the lines of NIST's proposal for [Consumer Software Cybersecurity Labeling](https://nvlpubs.nist.gov/nistpubs/CSWP/NIST.CSWP.02042022-1.pdf):
+In order to address these challenges and better inform users, Google and Apple have introduced new privacy labeling systems (very much along the lines of NIST's proposal) to help users easily understand how their data is being collected, handled, and shared, [Consumer Software Cybersecurity Labeling](https://nvlpubs.nist.gov/nistpubs/CSWP/NIST.CSWP.02042022-1.pdf). Their approaches can be seen at ...
```
```suggestion
In order to address these challenges and better inform users, Google and Apple have introduced new privacy labeling systems (very much along the lines of NIST's proposal) to help users easily understand how their data is being collected, handled, and shared, [Consumer Software Cybersecurity Labeling](https://nvlpubs.nist.gov/nistpubs/CSWP/NIST.CSWP.02042022-1.pdf). Their approaches can be seen at:
```

thornshadow99 commented:
Hi there,

Thanks for the complements! Inspired by your words, I have decided to make
more edits to your docs. I am a git beginner so please keep that in mind.

So I have forked the repository and made a branch (RLT-doc-changes). Then I
made edits to 0x03-Overview.md and committed them to my branch. Hopefully I
have done everything right. :)

Rob

On Wed, Feb 7, 2024 at 2:50 AM Carlos Holguera ***@***.***>
wrote:

> ***@***.**** approved this pull request.
>
> This is really good, thank you @thornshadow99
> <https://github.com/thornshadow99> and apologies on behalf of the project
> team for the delay! We've been focusing on the new proposals, core
> updates and changes <https://mas.owasp.org/news/> to be able to launch
> the new MASTG refactoring this year.
>
> We hope to have more contributions from you this year. Thank you very much!
> ------------------------------
>
> In Document/0x04i-Testing-User-Privacy-Protection.md
> <https://github.com/OWASP/owasp-mastg/pull/2463#discussion_r1367862627>:
>
> >      - **Privacy-by-Design** approach (Art. 25 GDPR, "Data protection by design and by default").
>      - **Principle of Least Privilege** ("Every program and every user of the system should operate using the least set of privileges necessary to complete the job.")
> -- **User Education**: Users need to be educated about their sensitive data and informed about how to use the application properly (to ensure secure handling and processing of their information).
> +    - **User Education**: Users need to be educated about their sensitive data and informed about how to use the application properly (to ensure secure handling and processing of their information).
>
> ⬇️ Suggested change
>
> -    - **User Education**: Users need to be educated about their sensitive data and informed about how to use the application properly (to ensure secure handling and processing of their information).
> +- **User Education**: Users need to be educated about their sensitive data and informed about how to use the application properly (to ensure secure handling and processing of their information).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OWASP/owasp-mastg/pull/2463#pullrequestreview-1691393801>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDMXOUJEGESTMV7IFBI7CWLYSMW5PAVCNFSM6AAAAAA6JYFBDSVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMYTMOJRGM4TGOBQGE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

cpholguera commented:
Thank you @thornshadow99! I really appreciate it. The steps you've taken sound exactly right. Once you've completed the next chapter, you can open a pull request from your fork to our master.

https://mas.owasp.org/contributing/2_Getting_Started/#contribute-offline

I recommend opening one pull request per chapter in your case. If you have any questions please let me know!

thornshadow99 commented:
Carlos,

I am trying to submit the pull request via github.dev and I can't seem to
create individual pull requests for each chapter. Do you know how to create
two separate pull requests for different chapters in github.dev? If not,
can I submit one request for two chapters this time and the next time just
submit a pull request for one chapter?

Rob

On Wed, Feb 7, 2024 at 3:07 PM Carlos Holguera ***@***.***>
wrote:

> Thank you @thornshadow99 <https://github.com/thornshadow99>! I really
> appreciate it. The steps you've taken sound exactly right. Once you've
> completed the next chapter, you can open a pull request from your fork to
> our master.
>
> https://mas.owasp.org/contributing/2_Getting_Started/#contribute-offline
>
> I recommend opening one pull request per chapter in your case. If you have
> any questions please let me know!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OWASP/owasp-mastg/pull/2463#issuecomment-1932782560>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDMXOUILDZF45WR2DQNDW2LYSPNHZAVCNFSM6AAAAAA6JYFBDSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSMZSG44DENJWGA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

thornshadow99 commented:
Also, it is worth noting that I took a section from 0x04 and moved it to
0x03 ("## What's Covered in the Mobile Testing Guide"). It's still in the
documentation, just moved to a different file.

Rob

On Wed, Feb 7, 2024 at 4:25 PM Robert Thornton ***@***.***>
wrote:

> Carlos,
>
> I am trying to submit the pull request via github.dev and I can't seem to
> create individual pull requests for each chapter. Do you know how to create
> two separate pull requests for different chapters in github.dev? If not,
> can I submit one request for two chapters this time and the next time just
> submit a pull request for one chapter?
>
> Rob
>
> On Wed, Feb 7, 2024 at 3:07 PM Carlos Holguera ***@***.***>
> wrote:
>
>> Thank you @thornshadow99 <https://github.com/thornshadow99>! I really
>> appreciate it. The steps you've taken sound exactly right. Once you've
>> completed the next chapter, you can open a pull request from your fork to
>> our master.
>>
>> https://mas.owasp.org/contributing/2_Getting_Started/#contribute-offline
>>
>> I recommend opening one pull request per chapter in your case. If you
>> have any questions please let me know!
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/OWASP/owasp-mastg/pull/2463#issuecomment-1932782560>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BDMXOUILDZF45WR2DQNDW2LYSPNHZAVCNFSM6AAAAAA6JYFBDSVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSMZSG44DENJWGA>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

cpholguera commented:
@thornshadow99 You can only submit one pull request per branch. It's okay to send the 2 chapters this time, especially since you've moved the content over, so it makes perfect sense.
